store/postgres/src/deployment_store.rs

```diff
@@ -920,6 +921,7 @@ impl DeploymentStore {
         conn: &PgConnection,
         site: Arc<Site>,
         block_ptr_to: BlockPtr,
+        firehose_cursor: Option<&str>,
```

Comment:
Should this still be an `Option`, given this is now using the empty string?

Reply:
Well, I used `""` at some place to "clear" the cursor, because right now `None` means do nothing (which is the case when using Ethereum RPC).

So I'm of the opinion that we should keep it like this.